Week1/homework/js-exercises/compareArray.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+var x = [1,2,3,4];
+
+var y = ['q','w','e','r','t','y','u'];
+
+console.log('The length of the array x is ' + x.length + " and the length of the array y is " + y.length);
+
+if (x.length === y.lenght) {
+    console.log("They are the same!");
+}
+else {console.log("Two different sizes");}
```

Comment:
1. 👍 for using triple equals
2. Please use `const` instead of `var` here.
3. There's a spelling mistake inside the `if` condition.
4. Minor indentation(spacing) problems with the `if-else` block. We should also strive for making our code readable due to which indentation becomes very important 🙂